project/api/serializers.py

```diff
@@ -22,25 +25,36 @@ class TerritorySerializer(ModelSerializer):
     """
     Serializes the Territory model as GeoJSON compatible data
     """
+    def to_internal_value(self, data):
+        ret = {}
+
+        # Update ret to include passed in data
+        for field, val in data.items():
+            if field == 'nation':
+                ret['nation'] = Nation.objects.get(pk=val)
+            if field != 'geo' and field != 'nation':
+                ret[field] = val
```

Comment:
What if I pass some data with field `foobar`? `ret['foobar']` will be set, right? No problem when writing that to the db?

Reply:
We'd have to create the foobar field beforehand. I'll test this when I get home as I'm not entirely sure what would happen, my guess would be it would still work with no foobar field being created.